test/test_align_cli.py

```diff
@@ -28,8 +28,14 @@ def test_invoke_align(self):
             [
                 "-i",
                 "-s",
-                "-C",
-                "-t",
+                "-o",
+                "vtt",
+                "-o",
+                "srt",
+                "-o",
+                "TextGrid",
+                "-o",
+                "eaf",
```

Comment:
Do we also support `-o vtt:srt:TextGrid:eaf`? I would prefer having that option when I'm using these interactively, it seems to me Click can support that. I don't know if `:` is the default separator. If click supports it, I might use the same thing for specifying multiple languages via `-l`.

Reply:
no I don't believe that's supported unfortunately. I'm not sure click supports that either...it seems odd. I see how you can do [fixed width options](https://click.palletsprojects.com/en/8.0.x/options/#:~:text=message%20and%20exit.-,Multi%20Value%20Options,-%C2%B6), but that's not what we want either.

Reply:
Well, it's quite easy to implement the way I did g2p_fallback, with a `.split(":")`.

Reply:
Another option, we can do it with a callback:
```
def joiner_callback(valid_values, joiner=":"):
    def _callback(ctx, param, value):
        results = [x for v in value for x in v.split(joiner)]
        for x in results:
            if x not in valid_values:
                raise click.BadParameter(f"'{x}' is not one of {', '.join(valid_values)}.")
        return results
    return _callback
...
@click.option(
    "-o",
    "--output-formats",
    multiple=True,
    callback=joiner_callback(SUPPORTED_OUTPUT_FORMATS),
    ...
```
And I'll be able to reuse it for `-l` too, with `callback=joiner_callback(LANGS)`.

I'm almost done implementing this change, so I'll clean it up and commit it.

Reply:
changes pushed.